api/Document.json

```diff
@@ -1632,11 +1632,10 @@
                 "notes": "For backwards compatibility, the <code>options</code> argument can be an object or a string with the custom element tag name, although the string version is deprecated."
               },
               "chrome_android": {
-                "version_added": false,
-                "notes": "For backwards compatibility, the <code>options</code> argument can be an object or a string with the custom element tag name, although the string version is deprecated."
+                "version_added": false
```

Comment:
I think `false` was misplaced here, based upon the other Chromium Android browsers, as well as a quick test in the latest Chrome Android.  Can we revert this and set it to `true`?